### PR TITLE
Misc last minute fixes

### PR DESCRIPTION
**Pull request by hsaliak**
Misc cosmetic fixes toward #258 

- Update navigation links to performance benchmarking.
- Added link to GA dashboard
- Remove redundant contribution guide link
- Point to github org as a landing point for "Fork on github"
- Updated a few supported platforms links.
- Used ```{{site.baseurl}}``` for many of the links
- Fixed Lisa's Hello gRPC! Tutorial  to use the right C# versions and with the Python API. 
- Fixed Android Java tutorial layout

### CODE REVIEW

**Comment by hsaliak** [comment]
@ejona86 for the styling and formatting
@nathanielmanistaatgoogle for the python tutorial in Hello gRPC!
@zinuga for the content changes.

File: docs/guides/hello.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ To install the gRPC runtime for your chosen language:
   </ul>
 
 <div id="java_install">
-<p>This example is also part of Java gRPC itself's build, so running it is a little different to creating and running a regular project. To install and build it follow the instructions in the <a href="https://github.com/grpc/grpc-java/tree/master/examples">Quickstart</a>.</p>
+<p>This example is also part of Java gRPC's build, so running it is a little different to creating and running a regular project. To install and build it follow the instructions in the <a href="https://github.com/grpc/grpc-java/tree/{{site.data.config.grpc_release_branch}}/examples">Quickstart</a>.</p>
```
Comment (ejona86) [inline docs/guides/hello.md]: This paragraph needs to be fixed a bit more. The example is no longer part of Java's build. I'll need to have a followup PR.

**Comment by ejona86** [comment]
@hsaliak LGTM

File: docs/guides/hello.md [hunk 2]
```diff
@@ -908,8 +909,8 @@ class GreeterClient {
   </div>
   <div id="python_connect">
 <p>The generated Python code has a helper function that creates a stub given a channel.</p>
-<pre>channel = implementations.insecure_channel('localhost', 50051)
-stub = helloworld_pb2.beta_create_Greeter_stub(channel)
+<pre>channel = grpc.insecure_channel('localhost', 50051)
```
Comment (nathanielmanistaatgoogle) [inline docs/guides/hello.md]: `grpc.insecure_channel` takes a single `target` parameter. In this case you want to pass [`'localhost:50051'`](https://github.com/grpc/grpc/blob/46585e23f14b0463c4b7a0d04d72d71228c/examples/python/helloworld/greeter_client.py#L40).

**Comment by nathanielmanistaatgoogle** [comment]
Rather than `$ ./run_codegen.sh`, `$ ./run_server.sh`, and `$ ./run_client.sh`, Python has [`$ python run_codegen.py`, `$ python greeter_server.py`, and `$ python greeter_client.py`](https://github.com/grpc/grpc.github.io/blob/ca64e4a045d75ca51b451224d53357d8ffbb75b2/docs/quickstart/python.md).

**Comment by hsaliak** [comment]
@nathanielmanistaatgoogle kindly take a look please. I have made the changes.

**Comment by zinuga** [comment]
LGTM

**Comment by nathanielmanistaatgoogle** [comment]
Where's the `$ ./run_server.sh` → `$ python greeter_server.py` change?

**Comment by hsaliak** [comment]
@nathanielmanistaatgoogle  PTAL

**Comment by nathanielmanistaatgoogle** [comment]
Luhguhtum but remember the "cd **grpc/**examples/python/helloworld" change to be done as well.

**Pull request merged**